src/FSharp.Build/CreateFSharpManifestResourceName.fs

```diff
@@ -30,37 +30,56 @@ type CreateFSharpManifestResourceName public () =
 
         let fileName, linkFileName, rootNamespace =
             match this.UseStandardResourceNames with
-            | true ->
-                fileName, linkFileName, rootNamespace
+            | true -> fileName, linkFileName, rootNamespace
             | false ->
-                let runningOnMono = 
+                let runningOnMono =
                     try
                         System.Type.GetType("Mono.Runtime") <> null
-                    with e -> 
-                        false  
-                let fileName = if not runningOnMono || fileName.EndsWith(".resources", StringComparison.OrdinalIgnoreCase) then fileName else Path.GetFileName(fileName)
-                let linkFileName = if not runningOnMono || linkFileName.EndsWith(".resources", StringComparison.OrdinalIgnoreCase) then linkFileName else Path.GetFileName(linkFileName)
-                fileName, linkFileName, "" 
+                    with
+                    | e -> false
 
-        let embeddedFileName = 
+                let fileName =
+                    if
+                        not runningOnMono
+                        || fileName.EndsWith(".resources", StringComparison.OrdinalIgnoreCase)
+                    then
+                        fileName
+                    else
+                        Path.GetFileName(fileName)
```

Comment:
These 1 --> 8 line expansions are quite hard on the eye

Reply:
@nofaj can you check these please?  The style guide says "If a condition is long, the then keyword is still placed at the end of the expression."

Reply:
I think I'm OK with altering the style guide.  But the penalty for long conditions is certainly striking.